import { AxiosError } from "axios";

export const errorInterceptor = (error: AxiosError) => {
  if (error.message === "Network Error") {
    return Promise.resolve(new Error("Erro de conex√£o."));
  }
  if (error.response?.status === 401) {
    // Do something
  }
  return Promise.reject(error);
};
